proto/validator/accounts/v2/web_api.pb.go

```diff
@@ -18,6 +14,9 @@ import (
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
+	io "io"
```

Comment:
Please revert these lines. Go import will fix this

Reply:
So the way i would do is, to open the file and move those 3 lines up to their original place?  it is a do not edit file.

Reply:
Or run `goimports` that will format these files for you